hack/make/release-deb

```diff
@@ -73,7 +73,10 @@ TreeDefault {
 };
 EOF
 
-for suite in $(exec contrib/reprepro/suites.sh); do
+for dir in contrib/builder/deb/${PACKAGE_ARCH}/*/; do
+	version="$(basename "$dir")"
```

Comment:
also the case in https://github.com/docker/docker/pull/21038/files#diff-c6c6bb0d2e164817a23d98d07109b25aR100 so its fine not updating it in this pr